src/components/breadcrumb/Breadcrumb.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
```

Comment:
ESLint rule warning: Using exported name 'styled' as identifier for default export. [import/no-named-as-default](https://github.com/import-js/eslint-plugin-import/blob/v2.29.1/docs/rules/no-named-as-default.md)

Reply:
yes, I saw, but I think it is really common to import styled-components from default. Are we changing all the imports  to styled-components to named import?

Reply:
In this repo all imports were changed to fix the lint rule, but as we decide, if we keep this default import I would create a specific rule to override the warning for `styled-components`

Reply:
I didn't know they were changed, sorry, anyway maybe we can talk about it